tests/test_checkout.py

```diff
@@ -32,12 +32,19 @@ def tearDownClass(cls):
     def setUp(self):
         dss.Config.set_config(dss.BucketConfig.TEST)
         self.s3_test_fixtures_bucket = get_env("DSS_S3_BUCKET_TEST_FIXTURES")
-        self.s3_test_checkout_bucket = get_env("DSS_S3_CHECKOUT_BUCKET_TEST")
-        self.s3_test_bucket = get_env("DSS_S3_BUCKET_TEST")
+        self.gs_test_fixtures_bucket = get_env("DSS_GS_BUCKET_TEST_FIXTURES")
```

Comment:
If these vars (s3_test_checkout_bucket and gs_test_checkout_bucket) are unused except for `get_test_fixture_bucket(..)`, I would just get rid of them, and do the get_env there.

Reply:
refactored